pallets/nfts/src/features/create_delete_collection.rs

```diff
@@ -110,6 +113,10 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 				ensure!(collection_details.owner == check_owner, Error::<T, I>::NoPermission);
 			}
 			ensure!(collection_details.items == 0, Error::<T, I>::CollectionNotEmpty);
+			ensure!(
+				AccountApprovals::<T, I>::get(collection, &collection_details.owner) == 0,
```

Comment:
Reassess based on understanding of collection approvals.

Reply:
Resolved in PR by refactoring the collection approval logic to allow any accounts instead of just collection owner: https://github.com/r0gue-io/pop-node/pull/395